### PR TITLE
rixmp package migration from `ixmp_dev`

### PR DESCRIPTION
**Pull request by adrivinca**
- `rixmp` R package, in _rixmp/source_ + gitignore in  _rixmp/source/inst/docum_ to allow the empty folder
- update `install.bat` that also install the rixmp and build the relative documentation IF R is installed
- remove `ixmp.R`
- update readme, gitignore, rst files for main documentation
- update tutorials
- update r package license and authors
- update `solve()` and `run_gams()` in R package to be independent from MESSAGEix

### CODE REVIEW

File: rixmp/source/R/class_ixmp.Platform.R [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        # Parameters
+        # ----------
+        # default : boolean, default True
+        # include only default model/scenario version (true) or all versions
+        # model : string
+        # the model name (optional)
+        # scen : string
+        # the scenario name (optional)
+
+        mod_scen_list = .jobj$getModelScenarioList(default, model, scenario)
```
Comment (danielhuppmann) [inline rixmp/source/R/class_ixmp.Platform.R]: this function was renamed to `getScenarioList()` on the Java API, this causes an error in the R transport tutorial 2

Comment (adrivinca) [reply rixmp/source/R/class_ixmp.Platform.R]: That's a big problem... I am trying to see what's wrong

**Comment by adrivinca** [comment]
The issue is now solved @danielhuppmann

**Comment by danielhuppmann** [comment]
the two tutorials now pass without errors! thanks @adrivinca !

**Pull request merged**